src/components/ha-labels-picker.ts

```diff
@@ -216,6 +216,7 @@ export class HaLabelsPicker extends SubscribeMixin(LitElement) {
     ha-input-chip {
       --md-input-chip-selected-container-color: var(--color, var(--grey-color));
       --ha-input-chip-selected-container-opacity: 0.5;
+      --_selected-outline-width: 1px;
```

Comment:
I think this should be `--md-input-chip-selected-outline-width`

Reply:
Let me test it. Am away from PC and will be able to check in a week.

Reply:
Agreed! Thanks for this addition.